examples/set-annotation/README.md

```diff
@@ -1,14 +1,14 @@
-# Annotate Config
+# Set Annotation
 
-The `annotate-config` KRM config function adds an annotation to resources.
+The `set-annotation` KRM config function adds an annotation to resources.
 
 ## Function invocation
 
 Get this example and try it out by running the following commands:
 
 ```sh
-kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/annotate-config .
-kpt fn run annotate-config/configs --fn-path annotate-config/functions
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/set-annotation .
+kpt fn run set-annotation/configs --fn-path set-annotation/functions
```

Comment:
this is still referring to `configs`. How come the tests are not failing?

Reply:
This is the readme file and it's irrelevant to the test. Will update it.

Reply:
done

Reply:
Thanks, incorrect documentation is a bug. At some point, we may want to use the md test harness used elsewhere (not urgent).